src/service/viewer-impl.js

```diff
@@ -1078,7 +1077,7 @@ export class Viewer {
     }
     this.messagingMaybePromise_.then(() => {
       if (this.messageDeliverer_) {
-        this.sendMessageUnreliable_(eventType, data, false);
+        this.messageDeliverer_(eventType, data, false);
```

Comment:
We should have just checked for `#messagingMaybePromise`, then called `#sendMessage`.